tests/test_api.py

```diff
@@ -42,6 +43,10 @@ def setUp(self):
         dummy_schedule_patcher.start()
         self.addCleanup(dummy_schedule_patcher.stop)
 
+        relative_dates_patcher = patch('edx_when.api._are_relative_dates_enabled', return_value=True)
+        relative_dates_patcher.start()
```

Comment:
Does this turn on the patcher for every test? Basically do this instead of having to write an override on every test for true?

Reply:
I did it this way for two reasons: (1) there is code right above this that does the same thing, so I felt like I was following form. And (2) I'm lazy -- if you override wih a decorator, you have to add a method argument for the mock to each test.